cmd/nmi/main.go

```diff
@@ -32,7 +33,7 @@ func main() {
 	if *debug {
 		log.SetLevel(log.DebugLevel)
 	}
-	log.Info("starting nmi process")
+	log.Info("Starting nmi process. Version: %v. Build date: %v", version.NMIVersion, version.BuildDate)
```

Comment:
Can we please have the same in mic main.go which prints the version there ?

Reply:
done